README.md

```diff
@@ -9,21 +9,17 @@
 ## Built With
 
 - CSS
+- Bootstrap
 - React
 - Redux
 
-## Screenshot of App
-<!-- <img width="763" alt="Leaderboard image" src="https://user-images.githubusercontent.com/58818795/132796576-8fab5b1c-fd2d-46c6-9f02-3d37898dba5e.png">
-
-<img width="763" alt="Leaderboard image" src="https://user-images.githubusercontent.com/58818795/132796694-8e77d562-6435-41f6-a354-6673e0ee3e00.png">
-
-<img width="763" alt="Leaderboard image" src="https://user-images.githubusercontent.com/58818795/132796737-9c16e670-90fb-4ca7-b7ca-98a409277a8d.png"> -->
-
-
 ## Video
 
-<!-- https://www.loom.com/share/86eeb7da0f744395a46ad417c81dad49 -->
+![link]https://www.loom.com/share/f89a97f5fa5f403dab880fd1b23ad4ee
+
+## Live Preview
 
+![link]https://enigmatic-island-12726.herokuapp.com/
```

Comment:
- [ ] The Live link is giving me this error. Please fix it or deploy it on[ netlify](https://www.netlify.com/) if the error still persists.

![Screenshot from 2021-09-24 23-51-43](https://user-images.githubusercontent.com/22434660/134724712-9e5dee79-050d-4b38-8c36-dbcd84d28548.png)